app/views/shared/_loss_report_modal.html.erb

```diff
@@ -0,0 +1,19 @@
+<div id="loss-report-modal" class="eui-modal-content" >
+  <a href="javascript:void(0);" class="modal-close float-r" id="close-loss-report-modal"><i class="fa fa-times"></i><span class="is-invisible">Close</span></a>
+  <div>
+    <h4>Conversion to UMM Format Required</h4>
+    <p>The native format of this collection is <%= options[:native_format].split('/').last.gsub('+xml','').upcase %>. Editing this record using the MMT will convert it to UMM-JSON, which may result in data loss. Select 'View Loss Report' to see how the conversion will affect this record.</p>
+    <p>
+      <%= link_to 'View Loss Report', loss_report_collections_path(options[:concept_id], format: 'json'), class: 'eui-btn eui-btn--steel-blue', target: :_blank %>
+
+      <% if current_provider?(options[:provider_id]) %>
+        <%= link_to 'Edit Collection', edit_collection_path(revision_id: options[:revision_id]), class: 'eui-btn--blue modal-close button-spacing' %>
+      <% elsif available_provider?(options[:provider_id]) %>
+        <%= link_to 'Edit Collection', '#', class: 'eui-btn--blue loss-report button-spacing' %>
+      <% end %>
+
+      <%= link_to 'Cancel', 'javascript:void(0);', class: 'eui-btn modal-close' %>
+      <%= link_to 'Invisible Link', '#not-current-provider-modal', class: 'display-modal not-current-provider is-invisible', id: 'loss-report-to-edit-collection', data: { provider: options[:provider_id], record_action: 'edit-collection' } %>
```

Comment:
Is this invisible link necessary? Could you add all this information to the Edit link directly?

Reply:
There were 2 things I considered when I was making this invisible link
1. closing one modal (the `loss-report-modal`) then opening another (the `not-current-provider-modal`) requires coffeescript so unfortunately I can just link directly to the next modal. There has to be a "close modal" action and then an "open modal" action in between. On top of that, the `not-current-provider-modal` is a complex modal to open (reference change_current_provider.coffee)
2. That being said, the `not-current-provider-modal` does a bit of event handling on its own, so I decided against copying all the code that is used to produce the `not-current-provider-modal` and pasting it into the handler that handles clicks on the `loss-report` class. From what I could tell the next easiest option was to have a hidden link (similar to the _not_current_provider_modal.html.erb partial)

Reply:
Also simply adding the classes from 'invisible link' to the Edit link unfortunately doesn't work

Reply:
I stand corrected, I did not properly add the classes from the invisible link to the Edit link. Thank you both for this suggestion